docs/features/traffic-management/mixed.md

```diff
@@ -0,0 +1,49 @@
+# Multiple Providers
+!!! note
+
+    Multiple trafficRouting is available since Argo Rollouts v1.1
+
+The usage of multiple providers tries to cover scenarios where, for some reason, we have to use
+different providers on North-South and West-East traffic routing or any other hybrid architecture that
+requires the use of multiple providers.
+
+## When to use multiple providers
+
+When you do not want:
```

Comment:
Is it possible to elaborate a bit more why it isn't recommended to use multiple providers in the cases bellow?

Reply:
noted

Reply:
What about now, looks better? Is that to be honest, I don't think about a suggestion on where to use it or not.. but mostly about examples on when you can use it if that's what you want.

Reply:
Nice. I understand much better what you mean by this section now. Tks!